doc/BMI_MODELS.md

```diff
@@ -332,7 +386,7 @@ A few other items of note:
 * configuration of `variables_names_map` maps a given variable to a variable name of the directly 
 * it is now possible to have an earlier nested module use as a provider (for one of its inputs) a later nested module, as long as a default value is configured
   * a collection of variable default values can be given in the formulation config at the top level by providing an entry in `default_output_values` with the variable's mapped configuration alias (or just variable name if it is unique) and the default value:
-```json
+```javascript
```

Comment:
Any particular reason for changing from `json` to `javascript`?

Reply:
That allows 
```javascript
//...
```
vs.
```json
//...
```
...somewhat subjective which is preferable. Neither really gives much useful code coloring.